descholar-ceo/find-flight-info
# User page

#### What does this PR ​ do​ ?
It allows user to go to his/her page after successful authentication

#### Description of Task to be completed?
* Create UserPage file
* Add functions in handlers file to handle the user page
* Add a directory called resources, to hold iata-airlines codes
* Add some stylings for the page

#### How should this be manually tested?
* Clone this repository by `git clone https://github.com/descholar-ceo/find-flight-info`
* Type these commands : 
    *  `cd find-flight-info` to go in the root directory of the project
    * `npm install` or `yarn add` to install all the packages
    * `npm start` or `yarn start`
* Then go in the browser, you will see a default opened page is signup
* Signup and and after signing up, you will be redirected to the userPage, where you will be able to 
some information to get the flight information

#### Any background context you want to provide?
* You have to install [nodejs](https://nodejs.org/en/) version 6 or higher prior to test this application

#### What are the relevant pivotal tracker stories?

#### Screenshots (​ if​ appropriate)
![03 userPage](https://user-images.githubusercontent.com/56886064/75100750-b9ffc500-55da-11ea-8553-e0b629190d54.png)
![04 userPage](https://user-images.githubusercontent.com/56886064/75100754-c2f09680-55da-11ea-976d-ffe32eb3f747.png)



#### Questions:
No Question
